src/core/ext/transport/binder/server/binder_server.cc

```diff
@@ -47,34 +47,32 @@ void RemoveEndpointBinder(const std::string& service) {
 }  // namespace experimental
 }  // namespace grpc
 
-grpc_core::Mutex* g_endpoint_binder_pool_mu = nullptr;
-absl::flat_hash_map<std::string, void*>* g_endpoint_binder_pool = nullptr;
-
-void grpc_endpoint_binder_pool_init() {
-  g_endpoint_binder_pool_mu = new grpc_core::Mutex();
-  g_endpoint_binder_pool = new absl::flat_hash_map<std::string, void*>();
-}
-
-void grpc_endpoint_binder_pool_shutdown() {
-  g_endpoint_binder_pool_mu->Lock();
-  delete g_endpoint_binder_pool;
-  g_endpoint_binder_pool_mu->Unlock();
-  delete g_endpoint_binder_pool_mu;
-}
+static absl::flat_hash_map<std::string, void*>* g_endpoint_binder_pool =
+    nullptr;
+static grpc_core::Mutex* g_endpoint_binder_pool_mu = new grpc_core::Mutex();
```

Comment:
I think we will want to delay the initialization of the mutex until we need it

something like

```
grpc_core::Mutex* GetBinderPoolMutex() {
  static grpc_core::Mutex* m = new grpc_core::Mutex();
  return m;
}
```

Reply:
Done.